Services/Component/classes/class.ilObjComponentSettingsGUI.php

```diff
@@ -517,17 +502,28 @@ function updatePlugin()
 		$pl = ilPlugin::getPluginObject($_GET["ctype"], $_GET["cname"],
 			$_GET["slot_id"], $_GET["pname"]);
 
-		$result = $pl->update();
-		
+		$this->update($pl);
+	}
+
+	/**
+	 * Execute update and ctrl reload
+	 *
+	 * @param ilPlugin 	$plugin
+	 *
+	 * @return null
```

Comment:
But im absolutely fine with removing the lines or leave it on null, just nitpicking :)